environment.yaml

```diff
@@ -3,6 +3,6 @@ channels:
   - conda-forge
 dependencies:
   - pip
-  - python=3.8
+  - python=3.10
```

Comment:
I checked out the branch and ran `make install` and now when I activate `dap_job_quality` and run `python --version` it says I'm on python 3.11.4... Don't know if this is a problem or not but it seemed surprising

Reply:
you did run make install! this is v weird...

Reply:
concerned lol as i think we should all be using the same python version given potential dependencies/conflicts - guess you could just downgrade?